Orange/widgets/regression/owunivariateregression.py

```diff
@@ -0,0 +1,234 @@
+from PyQt4.QtGui import QLayout
+
+from Orange.data import Table, Domain
+from Orange.regression.linear import (RidgeRegressionLearner, LinearModel,
+                                      LinearRegressionLearner, PolynomialLearner)
+from Orange.regression.base_regression import SklLearnerRegression
+from Orange.preprocess.preprocess import Preprocess
+from Orange.widgets import widget, settings, gui
+from Orange.widgets.utils import itemmodels
```

Comment:
Orange imports should be the last (after third party imports such as PyQt and numpy)
https://www.python.org/dev/peps/pep-0008/#imports